tools/add_font.py

```diff
@@ -23,6 +55,30 @@
                      ' for a -ext subset.')
 
 
+def family_url(name):
+  url_prefix = 'http://fonts.google.com/specimen/'
+  dl_name = name.replace(' ', '+')
+  return url_prefix + dl_name
+
+
+def family_exists(name):
+  """Check if a font family exists on fonts.google.com"""
+  url = family_url(name)
+  request = requests.get(url)
+  if request.status_code == 200:
+      return True
+  return False
+
+
+def _GetFamilyMetaData(name):
+  """Get family information from the frontend GF api"""
+  api_prefix = 'http://fonts.google.com/metadata/fonts/'
```

Comment:
@rsheeter Many families exist on fonts.google.com but don't have folders in this repository. 

hopefully, I will have all the missing fonts added to the repository this week. 

For the time being, I think we should leave this open. Once the repo and fonts.google.com are in sync, I can use your file method.

I'll also use gflags. Thank you for the review.